package/ubuntu/hypercontainer/debian/changelog

```diff
@@ -0,0 +1,5 @@
+hypercontainer (0.6.2-1) unstable; urgency=low
```

Comment:
I am not quite sure. If you want to build for ubuntu, maybe you should use the latest ubuntu distro code name instead of `unstable`. IIRC unstable is for new/updated regular debian packages.